composer/callbacks/checkpoint_saver.py

```diff
@@ -203,6 +203,8 @@ class CheckpointSaver(Callback):  # noqa: D101
         weights_only (bool): If ``True``, save only the model weights instead of the entire training state.
             This parameter must be ``False`` when using DeepSpeed. Default: ``False``.
 
+        save_planner (SavePlanner, optional): A :class:`~torch.distributed.checkpoint.planner.SavePlanner`. (default: ``None``)
```

Comment:
```suggestion
        save_planner (Optional[SavePlanner], optional): A :class:`~torch.distributed.checkpoint.planner.SavePlanner`. (default: ``None``)
```

Reply:
@dakinggg, no wait I think the original is correct

Reply:
wdym? you can specify `None` here

Reply:
I'm not sure what the docstring convention is, but the original followed how the other args were written.

Reply:
Actually this isn't relevant anymore after moving save_planner into fsdp_config